src/encoded/batch_download.py

```diff
@@ -174,6 +204,76 @@ def make_audit_cell(header_column, experiment_json, file_json):
     return ', '.join(list(set(data)))
 
 
+def _get_annotation_metadata(request, search_path, param_list):
```

Comment:
Where did this function come from?

Reply:
I made it. I could not cleanly augment the experiment metadata functionality with annotation. So, I made a function just for annotation, which is what you see.